model/project.rb

```diff
@@ -98,6 +99,26 @@ def current_invoice
     Invoice.new(project_id: id, content: content, begin_time: begin_time, end_time: end_time, created_at: Time.now, status: "current")
   end
 
+  def current_resource_usage(resource_type)
+    case resource_type
+    when "VmCores" then vms.sum(&:cores)
+    when "GithubRunnerCores" then github_installations.sum(&:total_active_runner_cores)
+    when "PostgresCores" then postgres_resources.flat_map { _1.private_subnet.vms.map(&:cores) }.sum
+    else
+      raise "Unknown resource type: #{resource_type}"
+    end
+  end
+
+  def effective_quota_value(resource_type)
+    default_quota = ProjectQuota.default_quotas[resource_type]
+    overide_quota_value = quotas_dataset.where(quota_id: default_quota["id"]).first&.value
```

Comment:
Is it a typo? I forgot to correct one 😄 
```suggestion
    override_quota_value = quotas_dataset.where(quota_id: default_quota["id"]).first&.value
```